resources/js/Pages/Results.vue

```diff
@@ -76,7 +76,7 @@
                         {{labels[item]}} ({{item}})
                     </a>
                 </h2>
-                <form @submit.prevent="send(item)">
+                <form @submit.prevent="send(String(item))">
```

Comment:
Maybe we should declare the type of the `item` at declaration time in the `v-for` instead of casting it evertime. I am not sure if this is possible, though.